src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F

```diff
@@ -232,25 +231,24 @@ subroutine cloudiness_from_MPAS(itimestep,configs,mesh,diag_physics,sfc_input,it
 
     case default
 
-    if (convection_scheme=="cu_grell_freitas") then
-       call mpas_pool_get_array(diag_physics,'cldfrac_cu',cldfrac_cu)
-       call mpas_pool_get_array(diag_physics,'qi_cu',qi_cu)
-       call mpas_pool_get_array(diag_physics,'qc_cu',qc_cu)
-       do j = jts,jte
-       do k = kts,kte
-       do i = its,ite
-          if (cldfrac_cu(k,i) .gt. 0.001) then
-             cldfrac_p(i,k,j) = min(1._RKIND,cldfrac_p(i,k,j) + cldfrac_cu(k,i))
-          endif
-          qccu_p(i,k,j) = qc_cu(k,i)
-          qicu_p(i,k,j) = qi_cu(k,i)
-       enddo
-       enddo
-       enddo
-    endif
-
  end select cld_fraction_select  
  
+ if (convection_scheme=="cu_grell_freitas") then
+    call mpas_pool_get_array(diag_physics,'cldfrac_cu',cldfrac_cu)
+    call mpas_pool_get_array(diag_physics,'qi_cu',qi_cu)
+    call mpas_pool_get_array(diag_physics,'qc_cu',qc_cu)
+    do j = jts,jte
+    do k = kts,kte
+    do i = its,ite
+       if (cldfrac_cu(k,i) .gt. 0.001) then
+          cldfrac_p(i,k,j) = min(1._RKIND,cldfrac_p(i,k,j) + cldfrac_cu(k,i))
```

Comment:
is the intention to always overwrite the mynn cldfrac calculated above when cu_grell_freitas?